<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dht11Widget</class>
 <widget class="QWidget" name="Dht11Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Config</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>max Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="maxTempSpinBox">
        <property name="suffix">
         <string>°C</string>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="minTempSpinBox">
        <property name="suffix">
         <string>°C</string>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>min Temperature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="alarmCheckBox">
        <property name="text">
         <string>Alarm</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>alarmCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTempSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alarmCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minTempSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
